src/db_queries/admin_partners_projects.ts

```diff
@@ -0,0 +1,56 @@
+import { Request, Response } from "express";
+import pg from "pg";
+import { Conn } from "../db/conn";
+import { Helpers } from "../utils/helpers";
+import PartnerProject from "../models/partner_project.model";
+
+const conn = new Conn();
+const pool = conn.pool;
+const helpers = new Helpers();
+
+export class AdminPartnersProjects {
+  constructor() {
+    helpers.autoBind(this);
+  }
+
+  async getAllProjectsOfOnePartner(
+    request: Request,
+    response: Response
+  ): Promise<void> {
+    const partnerId = request.params.partner_id;
+    const client = await pool.connect();
+    try {
+      const projects = await this.getAllProjectsOfOnePartnerFromDB(
+        partnerId,
+        client
+      );
+      response.status(200).json(projects);
+    } catch (error) {
+      response.status(400).send(error);
+    } finally {
+      client.release();
+    }
+  }
+
+  async getAllProjectsOfOnePartnerFromDB(
+    partnerId: string
+    ,client: pg.PoolClient
+  ): Promise<Array<PartnerProject>> {
+    const getPartnerProjectsQuery =`
```

Comment:
Let's have the SQL keywords in the query in uppercase so that there is consistency with the other queries.

Reply:
I usually follow that convention. But I saw this file for reference: src/db_queries/admin_partners_mentors.ts. This was exception not following the convention. Thanks for pointing. I will fix both.

Reply:
Thanks for switching the keywords to uppercase in the `admin_partners_mentors.ts` file as well, I had forgotten about that.